interface/forms/newpatient/common.php

```diff
@@ -86,16 +86,35 @@ function refreshIssue(issue, title) {
   s.options[s.options.length] = new Option(title, issue, true, true);
  }
 
+ <?php
```

Comment:
any reason why you are not instead using the automated script for this here:
``````
<?php validateUsingPageRules($_SERVER['PHP_SELF']);?>